coreneuron/network/partrans.cpp

```diff
@@ -41,40 +41,39 @@ void nrnmpi_v_transfer() {
     // gather the source values. can be done in parallel
     for (int tid = 0; tid < nrn_nthread; ++tid) {
         auto& ttd = transfer_thread_data_[tid];
-        auto& nt = nrn_threads[tid];
+        auto* nt = &nrn_threads[tid];
```

Comment:
just question - I assume this was just for consistency (and not hidden compiler issues), right?

Reply:
I was using `NrnThread* nt` in the `depend(inout: nt)` clauses, and the compiler rejected `depend(inout: &nt)` here IIRC (maybe it should be `*nt` everywhere, to be revisited when we re-add async stuff.

Reply:
Ah ok! make sense.